catalyst-gateway/bin/src/service/common/types/cardano/asset_value.rs

```diff
@@ -106,6 +106,17 @@ impl TryFrom<num_bigint::BigInt> for AssetValue {
     }
 }
 
+impl TryFrom<i128> for AssetValue {
```

Comment:
Probably don't need this, as we already have conversion from `BigInt` for this type.
See comment above about intermediate conversion to i128 being unnecessary.

That said, it doesn't hurt to keep it.  Just don't pre-convert `BigInt` to i128 before converting to `AssetValue`